bootloader/lkenv/lkenv_test.go

```diff
@@ -46,6 +46,7 @@ func Test(t *testing.T) { TestingT(t) }
 type lkenvTestSuite struct {
 	envPath    string
 	envPathbak string
+	testutil.BaseTest
```

Comment:
(nitpick^2) the usual pattern is to have the `testutil.BaseTest` as the first entry in the struct and (maybe) even a newline to make clear that "different" from the other entries

Reply:
(nitpick) I see that this was moved up here but in the other struts it's still at the bottom, can you please also update it there?